pkg/sync/syncer.go

```diff
@@ -84,12 +84,21 @@ func (p *ProgressCounts) LogResourcesProgress(ctx context.Context, resourceType
 func (p *ProgressCounts) LogEntitlementsProgress(ctx context.Context, resourceType string) {
 	entitlementsProgress := p.EntitlementsProgress[resourceType]
 	resources := p.Resources[resourceType]
+
+	l := ctxzap.Extract(ctx)
 	if resources == 0 {
+		if time.Since(p.LastEntitlementLog[resourceType]) > maxLogFrequency {
```

Comment:
Can you leave a comment here for why resources are 0.

Reply:
Done.